content/foundations/css-utilities/animations.mdx

```diff
@@ -5,46 +5,27 @@ description: Animations are reusable animation classes that you can use to empha
 
 <StorybookEmbed
   framework="css"
-  componentId="utilities-animation"
-  stories={{
-    'fade-in': {
-      name: 'Fade in'
-    },
-    'fade-out': {
-      name: 'Fade out'
-    },
-    'fade-up': {
-      name: 'Fade up'
-    },
-    'fade-down': {
-      name: 'Fade down'
-    },
-    'scale-in': {
-      name: 'Scale in'
-    },
-    'grow-x': {
-      name: 'Grow x'
-    },
-    pulse: {
-      name: 'Pulse'
-    },
-    hover: {
-      name: 'Hover animation'
-    },
-    rotate: {
-      name: 'Rotation'
-    }
-  }}
+  stories={[
```

Comment:
ℹ️ I updated the component API of `StorybookEmbed` so I had to update all its usages.